client/blocks/comments/post-comment-list.jsx

```diff
@@ -224,6 +227,22 @@ class PostCommentList extends React.Component {
 		);
 	};
 
+	renderCommentManageLink = () => {
+		const siteId = get( this.props, 'post.site_ID' );
+		const postId = get( this.props, 'post.ID' );
```

Comment:
Optional: We can use optional chaining and nullish coalescing to avoid needing the lodash get here.

```jsx
const siteId = this.props?.post?.site_ID ?? myDefaultValue;
const postId = this.props?.post?.ID ?? myOtherDefaultValue;
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator

Reply:
Looking at this again and I think that the `siteId` and `postId` can be accessed without either Lodash or above and still works fine! Is the approach that I've just committed okay? 

https://github.com/Automattic/wp-calypso/blob/c6eb351f8fd111e114b5a7a06d1ce67798e583d9/client/blocks/comments/post-comment-list.jsx#L481-L482